### PR TITLE
fixed navbar brand link

### LINKED ISSUE #43: Dead link in navbar brand
The ExtraMundane brand link is dead, needs to be updated to redirect to our home page.

### PR DESCRIPTION
**Pull request by Belax8**
closes #43

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
       <nav class="navbar navbar-default">
         <div class="container-fluid">
           <div class="navbar-header">
-            <button type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false" class="navbar-toggle collapsed"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a href="#" class="navbar-brand">ExtraMundane</a>
+            <button type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false" class="navbar-toggle collapsed"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button><a href="index.html" class="navbar-brand">ExtraMundane</a>
```
Comment (theflametrooper) [inline about.html]: You can change it to be: `href="\index.html"`

**Comment by theflametrooper** [comment]
@Belax8 You need to edit the jade and then compile it to html using `gulp`.

**Comment by Belax8** [comment]
@theflametrooper Ok, I think I did it right this time. Let me know if I need to fix anything else.

File: src/views/partials/header.jade [hunk 2]
```diff
@@ -7,13 +7,13 @@ header
           span.icon-bar
           span.icon-bar
           span.icon-bar
-        a.navbar-brand(href='#') ExtraMundane
+        a.navbar-brand(href='index.html') ExtraMundane
```
Comment (theflametrooper) [inline src/views/partials/header.jade]: ~~`index.html`~~ `/index.html`

**Comment by theflametrooper** [comment]
@Belax8 Please remove `main.css` from the changes and we should be good to merge!

**Comment by theflametrooper** [comment]
### **LGTM :+1:**
Thanks for your PR @Belax8 :tada:

**Pull request merged**